### PR TITLE
change max_size

### PR DESCRIPTION
**Pull request by BCecatto**
## Why this PR?
If try do a upload every time you get error from refile "too long" so I believe that better way is set this how nil, and if someone want change this I added this information in README.

### CODE REVIEW

File: lib/refile/cloudinary.rb [hunk 1]
```diff
@@ -24,7 +24,7 @@ class Cloudinary
     # @param [String] cloud_name The Cloudinary cloud name
     # @param [String] api_key The Cloudinary api key
     # @param [String] api_secret The Cloudinary api secret
-    def initialize(auth, max_size: 100)
+    def initialize(auth, max_size: nil)
```
Comment (pusewicz) [inline lib/refile/cloudinary.rb]: I just had a look at the [documentation](https://support.cloudinary.com/hc/en-us/articles/202520592-Do-you-have-a-file-size-limit-) and I think the default limit is 10 megabytes. Could you test that with `max_size: 10_485_760` and set it to as default?

Comment (BCecatto) [reply lib/refile/cloudinary.rb]: of course

File: lib/refile/cloudinary.rb [hunk 2]
```diff
@@ -24,7 +24,7 @@ class Cloudinary
     # @param [String] cloud_name The Cloudinary cloud name
     # @param [String] api_key The Cloudinary api key
     # @param [String] api_secret The Cloudinary api secret
-    def initialize(auth, max_size: 100)
+    def initialize(auth, max_size: 10_485_760)
```
Comment (pusewicz) [inline lib/refile/cloudinary.rb:27]: Perfect. As you have started working on tests, could you also add a test making sure that the default is of that value? Thanks!

Comment (BCecatto) [reply lib/refile/cloudinary.rb:27]: ok, I will do this.

**Comment by BCecatto** [comment]
Merge this first, because the other PR will broken test without the new max_size

**Pull request merged**